src/cityreader/cityreader.py

```diff
@@ -16,26 +24,32 @@
 # should not be loaded into a City object.
 cities = []
 
+
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
+  # Implement the functionality to read from the 'cities.csv' file
+  # For each city record, create a new City instance and add it to the
   # `cities` list
-    
+    with open('cities.csv', 'r', newline="") as csvfile:
+        csv_file = csv.reader(csvfile)
+        for c in csv_file:
+            cities.append(City(c[0], c[3], c[4]))
+
     return cities
 
+
 cityreader(cities)
 
 # Print the list of cities (name, lat, lon), 1 record per line.
 for c in cities:
-    print(c)
+    print(c.name, c.lat, c.lon)
```

Comment:
Close we want each record per line so you want to probably separate these by `\n`